tests/vm_test.go

```diff
@@ -2109,3 +2100,15 @@ func stopVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.Virtua
 
 	return vm
 }
+
+func createVMRunStrategyPatch(strategy v1.VirtualMachineRunStrategy) ([]byte, error) {
+	patches := patch.New(
+		patch.WithAdd("/spec/running", nil),
+		patch.WithAdd("/spec/runStrategy", string(strategy)),
+	)
+	patchData, err := patches.GeneratePayload()
```

Comment:
You can just do `return patches.GeneratePayload()`

Reply:
Thanks!